javascript/packages/fury/lib/gen/object.ts

```diff
@@ -80,11 +70,18 @@ class ObjectSerializerGenerator extends BaseSerializerGenerator {
   writeStmt(accessor: string): string {
     const options = this.description.options;
     const expectHash = computeStructHash(this.description);
-    const metaInformation = Buffer.from(computeMetaInformation(this.description));
+    // const metaInformation = Buffer.from(computeMetaInformation(this.description));
+    const fields = Object.entries(this.description).map(([key, value]) => {
+      return new FieldInfo(key, value.type);
+    });
+    const typeMetaBinary = new Uint8Array(TypeMeta.fromFields(256, fields).toBytes());
+    const typeMetaDeclare = this.scope.declare("typeMeta", `new Uint8Array([${typeMetaBinary.toString()}])`);
 
     return `
       ${this.builder.writer.int32(expectHash)};
-      ${this.builder.writer.buffer(`Buffer.from("${metaInformation.toString("base64")}", "base64")`)};
+      
+      ${this.builder.writer.buffer(typeMetaDeclare)};
```

Comment:
If it's SchemaConsistent， you can't write typeMeta for the same reason

Reply:
My solution is to check the fury modes  and write/read` typeMeta` only when the mode is `compatible` . Problem is every time i try to log the `fury.config` 
```ts
    console.log(`${JSON.stringify(this.builder.config)}`);
```
i get undefined